lib/route_translator/translator.rb

```diff
@@ -121,7 +121,10 @@ def self.default_locale?(locale)
     # segment is blank, begins with a ":" (param key) or "*" (wildcard),
     # the segment is returned untouched
     def self.translate_path_segment(segment, locale)
-      return segment if segment.blank? or segment.include?(":") or segment.starts_with?("(") or segment.starts_with?("*")
+      return segment if segment.blank?
+      named_param, hyphenized = segment.split('-'.freeze, 2) if segment.starts_with?(":".freeze)
+      return "#{named_param}-#{translate_path_segment(hyphenized.dup, locale)}" if hyphenized
+      return segment if segment.starts_with?("(".freeze) or segment.starts_with?("*".freeze) or segment.starts_with?(":".freeze)
```

Comment:
This line changed via #118 

https://github.com/enriclluelles/route_translator/pull/118/files#diff-6500de086a3ee34b17d3ed33ff3af908L124